src/engine_app_toolkit/include/iscene.h

```diff
@@ -88,6 +88,7 @@ class ENGINE_APP_TOOLKIT_API IScene
     virtual void activate();
     virtual void deactivate();
     virtual bool is_active() const;
+    virtual void set_active(bool active);
```

Comment:
1. Refactor this to remove "bool active" - its not needed.
2. Remove functions     virtual void activate(); virtual void deactivate();  and _is_activate member variable.
3. Refactor virtual bool is_active() const; to compare scene handles.